src/components/Footer.jsx

```diff
@@ -1,45 +1,64 @@
+      <div className="w-full md:w-1/6 flex flex-col items-center md:items-start">
+        <img src="images/symbol_enactus.png" alt="Logo" className="h-12" />
+        <div className="flex gap-3 text-gray-200 mt-4">
+          <a
+            href="https://in.linkedin.com/company/enactus-nsut"
+            target="_blank"
+          >
+            <LinkedInIcon className="text-[#0A66C2] scale-125" />
+          </a>
+          <a href="https://www.instagram.com/enac.nsut/" target="_blank">
+            <InstagramIcon className="text-pink-500 scale-110" />
+          </a>
+          <a href="https://www.x.com/EnactusNSUT/" target="_blank">
+            <XIcon className="text-black" />
+          </a>
+        </div>
       </div>
-    </div>
-    <div className='flex justify-between lg:justify-normal w-full'>
+      <div className="flex justify-between lg:justify-normal w-full">
         <div className="w-1/2 md:w-1/6 font-montserrat lg:mr-16">
-        <h1 className="font-bold mb-4">Explore</h1>
-        <ul className="space-y-2">
-            <li>Design</li>
-            <li>Prototyping</li>
-            <li>Development features</li>
-            <li>Design systems</li>
-            <li>Collaboration features</li>
-            <li>Design process</li>
-            <li>FigJam</li>
-        </ul>
+          <h1 className="font-bold mb-4">Connect with us</h1>
+          <ul className="space-y-2">
+            <li>
+              Netaji Subhash University of Technology,
+              <br />
+              Azad Hind Fauj Marg, Dwarka Sector-3, Dwarka, Delhi, 110078
+            </li>
+            <li>enactus@nsut.ac.in</li>
+          </ul>
         </div>
         <div className="w-1/2 md:w-1/6 font-montserrat">
-        <h1 className="font-bold mb-4 text-right md:text-right lg:text-left">Resources</h1>
-        <ul className="space-y-2 text-right md:text-right lg:text-left">
-            <li>Blog</li>
-            <li>Best practices</li>
-            <li>Colors</li>
-            <li>Color wheel</li>
-            <li>Support</li>
-            <li>Developers</li>
-            <li>Resource library</li>
-        </ul>
+          <h1 className="font-bold mb-4 text-right md:text-right lg:text-left">
+            Quick Links
+          </h1>
+          <ul className="space-y-2 text-right md:text-right lg:text-left">
+            <li>
+              <Link to="/">Home</Link>
+            </li>
+            <li>
+              <Link to="/Projects">Projects</Link>
+            </li>
+            <li>
+              <Link to="/Contact">Contact</Link>
+            </li>
+            <li>
+              <Link to="/Achievements">Achievements</Link>
+            </li>
+            <li>
+              <Link to="/Team">Team</Link>
+            </li>
+          </ul>
         </div>
-    </div>
-  </footer>
-}
\ No newline at end of file
+      </div>
+      <p className="text-[rgba(120,120,120,1)]">© 2024 Enactus NSUT</p>
```

Comment:
use javascript Date object to make it dynamic instead of coding the year yourself.